SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -7,18 +7,26 @@
 #include "EngineClasses/SpatialNetDriver.h"
 #include "EngineClasses/SpatialPackageMapClient.h"
 #include "EngineClasses/SpatialWorldSettings.h"
+#include "Interop/Connection/SpatialWorkerConnection.h"
 #include "LoadBalancing/AbstractLBStrategy.h"
+#include "Schema/ChangeInterest.h"
 #include "SpatialConstants.h"
 #include "SpatialGDKSettings.h"
 #include "Utils/Interest/NetCullDistanceInterest.h"
 
 #include "Engine/Classes/GameFramework/Actor.h"
 #include "Engine/World.h"
+#include "EngineClasses/SpatialReplicationGraph.h"
+#include "GameFramework/GameStateBase.h"
 #include "GameFramework/PlayerController.h"
+#include "GameFramework/PlayerState.h"
+#include "ReplicationGraph.h"
 #include "UObject/UObjectIterator.h"
+#include "Utils/MetricsExport.h"
```

Comment:
Needed?

Reply:
Yeah it's used below - perhaps it was in a follow up commit in the history rewrite